q2_feature_table/plugin_setup.py

```diff
@@ -607,3 +607,31 @@
     },
     citations=[citations['Hunter2007Matplotlib']]
 )
+
+plugin.methods.register_function(
+    function=q2_feature_table.split,
+    inputs={'table': FeatureTable[T1]},
+    parameters={
+        'metadata': MetadataColumn[Categorical],
+        'filter_empty_features': Bool
+    },
+    outputs=[
+        ('tables', Collection[FeatureTable[T1]])
+    ],
+    input_descriptions={
+        'table': 'The table to split.'
+    },
+    parameter_descriptions={
+        'metadata': 'A column defining the groups. Each unique value will '
+                    'define a split feature table.',
+        'filter_empty_features': 'If true, features which are not present in '
+                                 'a split feature table are dropped.',
+    },
+    output_descriptions={
```

Comment:
I wonder if it's worth including something in this description that clarifies that this output will be a directory containing all of the split feature tables - I know that this is the standard format for a collection, but I wonder if this might be confusing for people who either try to write these tables to an existing directory, or try to list out filenames for the number of split tables they're expecting.